akka-stream/src/main/boilerplate/akka/stream/FanInShape.scala.template

```diff
@@ -57,6 +57,21 @@ class UniformFanInShape[T, O](val n: Int, _init: FanInShape.Init[O]) extends Fan
   def in(n: Int): Inlet[T] = inArray(n)
 }
 
+class FanInShape1N[T0, T1, O](val n: Int, _init: FanInShape.Init[O]) extends FanInShape[O](_init) {
```

Comment:
Right. I had a need for this in some intermediate version which has been superseded since then.
But since I already wrote this shape I left it in in case you guys deem it of general utility.